tests/Whoops/Exception/FrameTest.php

```diff
@@ -103,6 +103,20 @@ public function testGetFileContents()
         $this->assertStringEqualsFile($data['file'], $frame->getFileContents());
     }
 
+    /**
+     * @covers Whoops\Exception\Frame::getFileContents
+     * @testWith ["[internal]"]
+     *           ["Unknown"]
+     * @see https://github.com/filp/whoops/pull/599
+     */
+    public function testGetFileContentsWhenFrameIsNotRelatedToSpecificFile($fakeFilename)
+    {
+        $data  = array_merge($this->getFrameData(), ['file' => $fakeFilename]);
```

Comment:
I guess we need php 5.x compat here

```suggestion
        $data  = array_merge($this->getFrameData(), array('file' => $fakeFilename));
```

Reply:
FYI, since you require php 5.5.9  or above, you should be fine.

> Following [] is supported in PHP >= 5.4: